relnotes/bigint.deprecation.md

```diff
@@ -0,0 +1,7 @@
+* `ocean.math.BigInt`
+
+  This module is deprecated in favor of `ocean.math.WideUint` because its
```

Comment:
Given the open source position on this, this release note seems a little bit too brutal. For example, `WideUint` is not a replacement for a `BigInt`. I'm not sure we can do anything about it, though.